app/templates/components/toc-section.hbs

```diff
@@ -1,9 +1,10 @@
 <button
   id={{this.id}}
   class="cp-Panel-toggle level-1 list-unstyled"
-  onclick={{action (mut this.open) (not this.open)}}
+  onclick={{fn (mut this.open) (not this.open)}}
```

Comment:
`not` is used, but it seems `ember-truth-helpers` is not defined as a dependency.

Reply:
@bertdeblock it seems to be functioning as expected. We can maybe take it out in follow up.

Reply:
@jaredgalanis Indeed, it's pulled in by `ember-styleguide`. I can create a new PR to add the dependency? Just to make sure we don't depend on `ember-styleguide` for `ember-truth-helpers` to be included.